src/auto/auto.yml

```diff
@@ -0,0 +1,42 @@
+# Orb Version 1.0.0
+
+version: 2.1
+description: An orb for working w/ intuit's auto
```

Comment:
This description is kind of vague, my suggestion- "Publish NPM packages and canary deployments with Intuit's Auto"

Reply:
```suggestion
description: Publish NPM packages and canary deployments with Intuit's Auto
```